docs/docs-ref-autogen/excel/excel.charttitle.yml

```diff
@@ -299,6 +312,19 @@ items:
       return:
         type:
           - number
+        description: |-
+
+          #### Examples
+
+          ```typescript
+          await Excel.run(async context => {

```

Comment:
`context` should have parens around it -> `await Excel.run(async (context) => {`